API.md

```diff
@@ -461,5 +461,23 @@ In this example, you can specify the custom path for storing images.
 In this example, you can freeze an image by specifying the selector path.
 
 
+## Page interactions
 
-
+```js
+(async () => {
+  await differencify
+    .init()
+    .newPage()
+    .goto('https://nimasoroush.github.io/differencify/')
+    .waitFor(2000)
+    .click('[id=js-repo-pjax-container] div nav span a')
+    .waitFor(2000)
+    .url()
+    .result((url) => {
+      expect(url).toContain('pulls');
+    })
```

Comment:
worth adding 

```
.screenshot()
.toMatchSnapshot()
```
as differencify is really about snapshot testing

Reply:
In this case the screenshot would change each time, as the url it navigates to is for a dynamic page (different PRs may be open each time it is ran). So the example is really showing that if no other solution is being used it also has the capabilities to test other things as well as visual tests.

Reply:
Can we click on `wiki` tab instead which is more or less static

Reply:
👍

Reply:
Awesome. Thanks for contribution